code/modules/religion/religion_sects.dm

```diff
@@ -193,3 +193,148 @@
 	adjust_favor(round(the_cell.charge/3000), L)
 	to_chat(L, "<span class='notice'>You offer [the_cell]'s power to [GLOB.deity], pleasing them.</span>")
 	qdel(I)
+
+
+//Fallen Sect
+
+/datum/religion_sect/fallen
+	name = "Sleeping Gods"
+	desc = "A sect that has fallen into dormancy."
+	convert_opener = "The gods have been silent. Where have you gone?!<br>You have no powers... But, in the darkest hour you will rise to save them all."
+	alignment = ALIGNMENT_GOOD //ancient defenders?
+	altar_icon_state = "sacrificealtar" //just no candles or the like, pretty dull altar
+	/// the current security level as a number, used to give out rewards
+	var/chaos_level = 0
+
+/datum/religion_sect/fallen/on_select()
+	RegisterSignal(SSdcs, COMSIG_GLOB_SECURITYLEVEL_CHANGED, .proc/alert_change)
+
+/datum/religion_sect/fallen/proc/alert_change(new_level)
+	var/increase = chaos_level < new_level ? TRUE : FALSE
+	var/godmessage = ""
+	var/wings = FALSE
+	chaos_level = new_level
+	switch(chaos_level)
+		if(SEC_LEVEL_GREEN)
+			godmessage = "[GLOB.deity] is silent, any murmurs of his power have vanished."
+		if(SEC_LEVEL_BLUE)
+			if(increase)
+				godmessage = "You feel a faint buzz of holy energies for just a moment."
+			else
+				godmessage = "You feel your powers fade as [GLOB.deity] returns to it's slumber..."
+		if(SEC_LEVEL_RED)
+			if(increase)
+				godmessage = "You feel your powers surge as [GLOB.deity] awakens!"
+			else
+				godmessage = "[GLOB.deity] is satisfied with the end of the delta event. The wings are your reward."
```

Comment:
what if you didn't get wings, due to being an ethereal or something?

Reply:
![image](https://user-images.githubusercontent.com/40974010/76611616-0d797a80-64d8-11ea-9993-0aad135fbc1c.png)
what are you talking about?

Reply:
And does this properly give dragon wings to lizards or do they get angel wings because god dont care about your species

Reply:
they get dragon wings. if they have a unique wing sprite, they get it!

Reply:
Cool, we should remove the species restriction on flight potions then

Reply:
what about moths?